app/decorators/registrations_controller_decorator.rb

```diff
@@ -0,0 +1,10 @@
+class RegistrationsControllerDecorator
```

Comment:
Decorators are not mentioned in the writeup.  There should be a comment here saying what they are for.